README.md

```diff
@@ -45,4 +45,4 @@ See also the ISAAC manual ([Dutch](https://www.nwo.nl/sites/nwo/files/media-file
 
 ## Dependencies
 
-This code uses [Citation.js](https://citation.js.org), license MIT.
+This code uses [Citation.js](https://citation.js.org), license MIT. Information is collected from CrossRef, Google Books, and Unpaywall.
```

Comment:
DOI uses Crossref, DataCite and mEDRA. ISBN uses Google Books and OpenLibrary. PubMed uses PubMed.